lightkurve/lightcurve.py

```diff
@@ -312,13 +312,18 @@ def flatten(self, window_length=101, polyorder=2, return_trend=False,
             # Reduce `window_length` and `polyorder` for short segments;
             # this prevents `savgol_filter` from raising an exception
             # If the segment is too short, just take the median
+
             if np.any([window_length > (h - l), (h - l) < break_tolerance]):
                 trend_signal[l:h] = np.nanmedian(lc_clean.flux[l:h])
             else:
-                trend_signal[l:h] = signal.savgol_filter(x=lc_clean.flux[l:h],
-                                                         window_length=window_length,
-                                                         polyorder=polyorder,
-                                                         **kwargs)
+                # Scipy outputs a warning here that is not useful, will be fixed in version 1.2
+                with warnings.catch_warnings():
+                    warnings.simplefilter('ignore')
```

Comment:
Let's specifically ignore `FutureWarning`, in case other warnings happen some day which we do want to know about:
```suggestion
                    warnings.simplefilter('ignore', FutureWarning)
```